packages/jaeger-ui-components/src/uiElementsContext.tsx

```diff
@@ -127,87 +59,10 @@ export const UIMenuItem = function UIMenuItem(props: MenuItemProps) {
   );
 };
 
-export type ButtonHTMLType = 'submit' | 'button' | 'reset';
-export type ButtonProps = {
-  children?: React.ReactNode;
-  className?: string;
-  htmlType?: ButtonHTMLType;
-  icon?: string;
-  onClick?: React.MouseEventHandler<HTMLButtonElement>;
-  disabled?: boolean;
-};
-
-export const UIButton = function UIButton(props: ButtonProps) {
-  return (
-    <GetElementsContext>
-      {(elements: Elements) => {
-        return <elements.Button {...props} />;
-      }}
-    </GetElementsContext>
-  );
-};
-
-export type DividerProps = {
-  className?: string;
-  type?: 'vertical' | 'horizontal';
-};
-
-export const UIDivider = function UIDivider(props: DividerProps) {
-  return (
-    <GetElementsContext>
-      {(elements: Elements) => {
-        return <elements.Divider {...props} />;
-      }}
-    </GetElementsContext>
-  );
-};
-
-export type InputProps = {
-  autosize?: boolean | null;
-  placeholder?: string;
-  onChange: (value: React.ChangeEvent<HTMLInputElement>) => void;
-  suffix: React.ReactNode;
-  value?: string;
-};
-
-export const UIInput: React.FC<InputProps> = function UIInput(props: InputProps) {
-  return (
-    <GetElementsContext>
-      {(elements: Elements) => {
-        return <elements.Input {...props} />;
-      }}
-    </GetElementsContext>
-  );
-};
-
-export type InputGroupProps = {
-  className?: string;
-  compact?: boolean;
-  style?: React.CSSProperties;
-  children?: React.ReactNode;
-};
-
-export const UIInputGroup = function UIInputGroup(props: InputGroupProps) {
-  return (
-    <GetElementsContext>
-      {(elements: Elements) => {
-        return <elements.InputGroup {...props} />;
-      }}
-    </GetElementsContext>
-  );
-};
-
 export type Elements = {
-  Popover: React.ComponentType<PopoverProps>;
-  Tooltip: React.ComponentType<TooltipProps>;
-  Icon: React.ComponentType<IconProps>;
   Dropdown: React.ComponentType<DropdownProps>;
```

Comment:
Should we remove these components as well in favor of grafana UI ones?

Reply:
Hmm right I wanted to wait for the span links because I thought it might affect that part of the code. But that PR is merged already so I can check if these can also be removed.

Reply:
So I removed those, but right now there is no ready component to use there. There is no regression as those components weren't implemented anyway. So I just left some todos in places where we should probably implement something for a list of things (like multiple references/links etc). So for now I assume this is fine and this PR can still be referenced in the future to see how it was implemented before.